.circleci/config.yml

```diff
@@ -0,0 +1,27 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/circleci-demo-cirpython-django
+    docker:
+      - image: circleci/python:3.6.4
+    steps:
+      - checkout
+      - restore_cache:
+          key: deps1-{{ .Branch }}-{{ checksum "requirements.txt" }}
+      - run:
+          command: |
+            python3 -m venv venv
+            . venv/bin/activate
+            pip install -r requirements.txt
```

Comment:
use the command `make requirements` from `Makefile`

Reply:
done.